app/controllers/results_controller.rb

```diff
@@ -154,6 +154,7 @@ def show
           data[:notes_count] = submission.grouping.notes.count
           data[:num_marked] = assignment.get_num_marked(current_user.admin? ? nil : current_user.id)
           data[:num_assigned] = assignment.get_num_assigned(current_user.admin? ? nil : current_user.id)
```

Comment:
If `num_assigned` is not being used please remove it here as well.
If `get_num_assigned` is not being used anywhere else please remove that function as well.